tests/e2e-playwright/tests/conftest.py

```diff
@@ -217,23 +221,29 @@ def user_name(request: pytest.FixtureRequest, auto_register: bool, faker: Faker)
 @pytest.fixture
 def user_password(
     request: pytest.FixtureRequest, auto_register: bool, faker: Faker
-) -> str:
+) -> SecretStr:
```

Comment:
I believe when @sanderegg introduced this, it was intentionally done to make it easier for us to debug by logging the username and password.

Reply:
I don't see any advantage in that. In this case, just hide it as you intended.